playbooks/provision-idm/dns.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: 'Copying state from idm hosts'
+  hosts: dns-server
+  tasks:
+   - set_fact: 
+       dns_records: "{{ hostvars[groups['idm'][0]]['dns_records'] }}" 
+       
+- name: 'Add DNS records for IdM'
```

Comment:
Can this be updated to calll the `update-dns-records.yml` playbook rather than accessing the role directly?